hydra_redis/collections_endpoint.py

```diff
@@ -16,8 +17,19 @@ def fetch_data(self, new_url):
         :param new_url: url for fetching the data.
         :return: loaded data.
         """
-        response = urllib.request.urlopen(new_url)
-        return json.loads(response.read().decode('utf-8'))
+        try:
+            response = urllib.request.urlopen(new_url)
+        except HTTPError as e:
+            print('Error code: ', e.code)
```

Comment:
These prints should be using a logger from `logging` library: https://docs.python.org/3/howto/logging-cookbook.html